auxiliary_tools/metrics_checker.py

```diff
@@ -0,0 +1,60 @@
+#!usr/bin/env python
+from __future__ import division, print_function
```

Comment:
Aren't we using `python3` now? I don't think we should still have `__future__` imports then.